perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -830,6 +830,22 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Forces group structure synchronization.
+	 *
+	 * @param group int Group <code>id</code>
+	 */
+	forceGroupStructureSynchronization {
+
+		@Override
+		public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
+
```

Comment:
Should be here `ac.stateChangingCheck();` @zlamalp?

Reply:
If method changes state (which it does), then yes.